lib/web_ui/test/text_editing_test.dart

```diff
@@ -450,8 +450,8 @@ Future<void> testMain() async {
       sendFrameworkMessage(codec.encodeMethodCall(clearClient));
     }
 
-    String getEditingInputMode() {
-      return textEditing!.strategy.domElement!.getAttribute('inputmode')!;
+    String? getEditingInputMode() {
+      return textEditing!.strategy.domElement!.getAttribute('inputmode');
```

Comment:
`.inputMode`? https://api.dart.dev/stable/2.9.1/dart-html/Element/inputMode.html (non-nullable, wow, should we report this as a dart:html bug?)

Reply:
Nice find! I didn't know about that one, but it definitely should be nullable.

https://github.com/dart-lang/sdk/issues/49014